cupy/core/_reduction.pyx

```diff
@@ -180,7 +180,7 @@ cdef Py_ssize_t _get_contiguous_size(
     return contiguous_size
 
 
-cpdef (Py_ssize_t, Py_ssize_t, Py_ssize_t) _get_block_specs(  # NOQA
+cpdef(Py_ssize_t, Py_ssize_t, Py_ssize_t) _get_block_specs(  # NOQA
```

Comment:
Is this a typo? I think we need the space...

Reply:
Thanks, I fixed.